projects/spanner_emulator/project.yaml

```diff
@@ -0,0 +1,3 @@
+homepage: "https://github.com/googleinterns/cloud-spanner-emulator-fuzzing"
+language: c++
+primary_contact: "evmaus@google.com"
```

Comment:
Can we add some auto_ccs as well?

I'd suggest we start with you and Sneha as auto-ccs.

Reply:
fixed